### ISSUE TITLE
png pre-commit hook: bug when there is a description

### ISSUE THREAD

**Issue by lemairecarl** [vitalab/vitalab.github.io#452]
I just tested
```
bash utils/check_png_images.sh test.md
```
for the following `test.md`:
```
![image description](yolo.png)
```
And the script did not notify the presence of the png. If I remove the description (i.e. use `![](...`), it works.

@jhlegarreta

**Comment by jhlegarreta** [comment]
Thanks for reporting. I will need to put off having a look at this for a couple of months due to my workload.

Thanks for your patience.

**Comment by lemairecarl** [comment]
Agreed, you have better to do.

I found a solution using https://www.regexplanet.com/advanced/perl/index.html for checking.

Fixed in 06b9cdee0f0a750720ed4bfa5a0bdf008c3919d7.
